### ISSUE TITLE
Using default load_mask() when training Mask R-CNN

### ISSUE THREAD

**Issue by rvignav** [nageshsinghc4/Brain-Tumor-Detection-using-Mask-R-CNN#1]
When I run your code as is, I get a seemingly infinite loop of this message when training the model:
```
WARNING:root:You are using the default load_mask(), maybe you need to define your own one.
```
What must I do to fix this? I haven't changed your code at all besides fixing some imports and downgrading TF to 1.x and Keras to 2.1.3.

Thanks @nageshsinghc4

**Comment by fizaict** [comment]
@rvignav were you able to fix this?

**Comment by nageshsinghc4** [comment]
@rvignav I think its related to your TensorFlow and Keras version. Try tweeting them a bit.

**Comment by rvignav** [comment]
Hi @fizaict and @nageshsinghc4, thanks for your response. I have decided to use a different Mask-RCNN implementation, so this error need not be open anymore. However, if I do come back to this, I will definitely reach out to you. Thank you!
